code/obj/artifacts/artifact_items/teleport_wand.dm

```diff
@@ -85,4 +85,23 @@
 		for(var/atom/X in T.contents)
 			if (X.density)
 				return FALSE
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if(H.shoes?.magnetic)
+				boutput(user, "<span class='alert'><b>The magnetic locks on [H.shoes] overload!</b></span>")
+				playsound(H, pick('sound/impact_sounds/Flesh_Stab_1.ogg','sound/impact_sounds/Metal_Clang_1.ogg','sound/impact_sounds/Slimy_Splat_1.ogg','sound/impact_sounds/Flesh_Tear_2.ogg','sound/impact_sounds/Slimy_Hit_3.ogg'), 30)
```

Comment:
why is this being done here rather than where `can_teleport_here()` is called?

Reply:
artifact other effect ideas I had in mind, will change